docs/loading/Spark-connector-starrocks.md

```diff
@@ -163,15 +165,42 @@ Construct data in memory and load data into the StarRocks table.
     // 2. Write to starrocks with the format "starrocks",
     // and replace the options with your own.
     df.write.format("starrocks")
-        .option("starrocks.fe.http.url", "127.0.0.1:8038")
-        .option("starrocks.fe.jdbc.url", "jdbc:mysql://127.0.0.1:9038")
+        .option("starrocks.fe.http.url", "127.0.0.1:8030")
+        .option("starrocks.fe.jdbc.url", "jdbc:mysql://127.0.0.1:9030")
         .option("starrocks.table.identifier", "test.score_board")
         .option("starrocks.user", "root")
         .option("starrocks.password", "")
         .mode("append")
         .save()
     ```
 
+For python, run the following codes in `pyspark`:
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/44db077b-c6b0-42c6-a0cf-a273f68b629e)
Indent this part more to be aligned with the above "For scala, run the following codes in `spark-shell`:" part.